staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -204,17 +203,17 @@ func (c *delegatingCRConverter) ConvertToVersion(in runtime.Object, target runti
 
 	desiredAPIVersion := toGVK.GroupVersion().String()
 	if !c.validVersions[toGVK.GroupVersion()] {
-		return nil, fmt.Errorf("request to convert CR to an invalid group/version: %s", desiredAPIVersion)
+		return nil, runtime.NewNotRegisteredGVKErrForTargetWithContext("", toGVK, target)
```

Comment:
this isn't going to make sense... it will say "$group/$version is not suitable for converting to $group/version", right? is there a test that exercises this code path where we can see the message that gets produced?